### PR TITLE
Tests: Modification to the swarm tests

### PR DESCRIPTION
**Pull request by GabyCT**
- Integration of a function that will verify that swarm is not
running previously or finishing it correctly
- Use of the nginx image for swarm testing
- Check that the replicas' names are different test disable

Signed-off-by: Cervantes Tellez, Gabriela <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

**Comment by chavafg** [comment]
qa-failed

**Comment by grahamwhaley** [comment]
lgtm
in theory - let's see why the qa-failed etc.

File: tests/integration/docker/swarm.bats [hunk 1]
```diff
@@ -68,6 +80,7 @@ setup() {
 }
 
 @test "check that the replicas' names are different" {
+	skip "this test is not working properly (see https://github.com/01org/cc-oci-runtime/issues/578)
```
Comment (chavafg) [inline tests/integration/docker/swarm.bats]: quotes are missing at the end of the skip message.

**Comment by gorozco1** [comment]
lgtm

**Comment by chavafg** [comment]
qa-passed

**Pull request merged**